README.md

```diff
@@ -1,2 +1,96 @@
 # rotation-scheduler
```

Comment:
I know the actual repo name needs to be hyphenated, but in common usage could this be called "Rotation Scheduler"?

Reply:
Done.